pkgs/build-support/build-graalvm-native-image/default.nix

```diff
@@ -50,6 +49,8 @@ stdenv.mkDerivation (args // {
     runHook postInstall
   '';
 
+  disallowedReferences = [ graalvmDrv ];
```

Comment:
Just want to make sure that we don't have this issue anymore once we update the GraalVM.